### ISSUE TITLE
Numpy AttributeError from gdt/core/plot/model.py

### ISSUE THREAD

**Issue by rachel-hamburg** [USRA-STI/gdt-core#74]
When calling ModelFit() from `gdt/core/plot/model.py`, I received this error:
```
Traceback (most recent call last):
  File "/Users/rachel/Work/GBM/Spectral-Catalog/spec-catalog/ServerCat/Server_catalog_grb.py", line 227, in <module>
    spectral_analysis(par_in, fp, outdir=output_file_path, logger=None, verbose=False)
  File "/Users/rachel/Work/GBM/Spectral-Catalog/spec-catalog/ServerCat/spectral_analysis/spec_cat.py", line 226, in spectral_analysis
    plot_modelfits(fit_results_flnc, 'fluence', outdir=plot_dir)
  File "/Users/rachel/Work/GBM/Spectral-Catalog/spec-catalog/ServerCat/spectral_analysis/spec_cat_tools.py", line 194, in plot_modelfits
    modelplot = ModelFit(fitter=fit)
  File "/Users/rachel/venv/gdt-devel/lib/python3.10/site-packages/gdt/core/plot/model.py", line 60, in __init__
    warnings.filterwarnings("ignore", category=np.VisibleDeprecationWarning)
  File "/Users/rachel/venv/gdt-devel/lib/python3.10/site-packages/numpy/__init__.py", line 414, in __getattr__
    raise AttributeError("module {!r} has no attribute "
AttributeError: module 'numpy' has no attribute 'VisibleDeprecationWarning'
```

In line 60 of `model.py`, I see 
```
warnings.filterwarnings("ignore", category=np.VisibleDeprecationWarning)
```

I am using numpy 2.1.2 but this issue can be fixed by setting a lower numpy version in `requirements.txt`. When using the newer numpy version, one needs to replace the above line with
```
warnings.filterwarnings("ignore", category=np.exceptions.VisibleDeprecationWarning)
```
since in newer versions of numpy `VisibleDeprecationWarning` has been moved to the numpy.exceptions module.

**Comment by AdamGoldstein-USRA** [comment]
@rachel-hamburg This should be fixed in PR #71.  Although we have not issued an official release containing this fix yet, you should be able to install from the main branch to see if PR #71 fixes your issue.

**Comment by rachel-hamburg** [comment]
Yes this worked, thank you
